py/client/pydeephaven/session.py

```diff
@@ -181,21 +217,24 @@ def close(self) -> None:
     def release(self, ticket):
         self.session_service.release(ticket)
 
-    def _parse_script_response(self, response):
-        if response.changes.created:
-            for t in response.changes.created:
+    def _parse_fields_change(self, fields_change):
+        if fields_change.created:
+            for t in fields_change.created:
                 t_type = None if t.typed_ticket.type == '' else t.typed_ticket.type
                 self._tables[t.field_name] = (t_type, Table(session=self, ticket=t.typed_ticket.ticket))
 
-        if response.changes.updated:
-            for t in response.changes.updated:
+        if fields_change.updated:
+            for t in fields_change.updated:
                 t_type = None if t.typed_ticket.type == '' else t.typed_ticket.type
                 self._tables[t.field_name] = (t_type, Table(session=self, ticket=t.typed_ticket.ticket))
 
-        if response.changes.removed:
-            for t in response.changes.removed:
+        if fields_change.removed:
+            for t in fields_change.removed:
                 self._tables.pop(t.field_name, None)
 
+    def _parse_script_response(self, response):
+        self._parse_fields_change(response.changes)
```

Comment:
I think we may be doubling up on our response handling for code we execute via scripts, in the case where we are also subscribing to list fields. We may want to ignore list_field changes that apply to the global script scope (FieldInfo.application_id == "scope"), or choose to only handle list_field changes (ignore script responses) when they are active.

The problem w/ the former is if we ignore the "scope" changes for list_field, that will miss updates executed in the web UI instead of via the python client.

The problem w/ the latter is that updates to the fields will be async with the response to script execution response.

Reply:
I decided to go with the former option, because otherwise the user can't rely on any of the tables they create in a script being usable until after some arbitrary delay, which seems like a significant usability problem.
Plus, ignoring global-script-scope changes doesn't harm existing functionality.